docs/architecture/adr-008-CERT-group.md

```diff
@@ -0,0 +1,92 @@
+# ADR 008: Computer Emergency Response Team (CERT) Group
+
+## Changelog
+
+- 2019 Jul 31 : Initial Conception
+
+## Context
+
+In order to reduce the number of parties involved with handling sensitive
+information in an emergency scenario, we propose the creation of a
+specialization group named The Computer Emergency Response Team (CERT).
+Initially this group's role is intended to serve as coordinators between the
+validators, the developers, and other members of the wider community.
+Additionally, a special privilege is proposed for the CERT group: the capacity
+to "circuit-break" (aka. temporarily disable)  a particular transaction path. 
+
+In the future it is foreseeable that the community may wish to expand
+the roles of CERT with further responsibilities such as the capacity to
+"pre-approve" a security update hard-fork on behalf of the community prior to a
+full community wide vote whereby the sensitive information would be revealed
+prior to a vulnerability being patched on the live network.  
+
+## Decision
+
+The CERT group is proposed to include an implementation of a `SpecializationGroup`
+as defined in [ADR 007](./adr-007-specialization-groups.md). This will include the 
+implementation of: 
+ - continuous voting
+ - slashing due to breach of soft contract
+ - revoking a member due to breach of soft contract
+ - compensation stipend from the community pool 
+
+This system necessitates the following new parameters: 
```

Comment:
it would be cool to add the reasoning for the suggested values

Reply:
I have no good reasonings right now, we should add that as an addendum to this ADR before we launch specialization groups, and after we've had some more discussions on this I reckon